vr-wordle/src/Components/Pillars.js

```diff
@@ -4,28 +4,28 @@ export default function Pillars(props) {
   return (
     <>
       <Column
-        position={[-1.25, 0, 0.4]}
-        guessIndex={4}
+        position={[-1.25, 0.2, -0.9]}
```

Comment:
have you considered assigning a variable for position so you don't have to change individually?